Library/Formula/vault.rb

```diff
@@ -0,0 +1,35 @@
+require "language/go"
+
+class Vault < Formula
+  homepage "https://vaultproject.io/"
+  version "0.1.0"
+  url "https://github.com/hashicorp/vault.git"
+
+  head "https://github.com/hashicorp/vault.git"
+
+  depends_on "go" => :build
+
+  def install
+    gopath = buildpath/"../gopath"
+
+    mkdir_p gopath
+    ENV["GOPATH"] = gopath
+    ENV.prepend_create_path "PATH", gopath/"bin"
+
+    system "go", "get", "github.com/tools/godep"
```

Comment:
Instead of running `go get`, can you stage these as `go_resource`s? If you take a look at deisctl you can find a formula that does this, including godep.